.travis.yml

```diff
@@ -6,10 +6,16 @@ sudo: required
 before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+install:
+  - npm install -g codecov
+  - npm install
 before_script:
   - npm run download-sync-client
 script:
   - npm run testsuite
+after_success:
```

Comment:
i don't think this is what you want, since it will run the coverage scripts once for every successful test in the suite. if all else fails you can add a new TEST_DIR value for it and add it as the last "test" in the testsuite, similar to what is done for TEST_DIR=lint